### DISCUSSION TITLE
Google Analytics Setup

### DISCUSSION THREAD

**Discussion by daCking15** [mozilla/hubs#3101] [General Questions & Discussion] [answered]
I am trying to get Google Analytics setup with my custom client. After setting up things in Google Analytics, I acquired the tracker ID (UA-###), but putting that into the admin panel did nothing. I even used the Google Tag chrome extension to see if it picked up anything. I then tried putting the script provided by Google into the extra script fields in the admin panel. After doing this, the chrome extension did pick up the tracker, but CORS errors prevented it from sending any data. It now seems that I am left with no other choice but to update the CORS origins. I have been cautious of doing this because of past server issues due to this field explained here: [Locked Out of Server after Editing CORS origins](https://github.com/mozilla/hubs/discussions/2890) Additionally, while the reply to this link explains the default CORS Origins, I am currently unaware of my internal domain and could use some help figuring it out (I am awaiting a reply from the person who instantiated the EC2 instance) and also, since our custom client is deployed on a subdomain, I wasn't sure if that would be the "main domain" listed in the reply. Thanks.

**Comment by robinkwilson** [comment, marked as answer]
The subdomain would be the "main domain", but as you're configuring CORS, there's no harm in adding the root domain as well.

You can find the "internal domain" if you look in the browsers Developer Tools > Network tab. There should be a domain you see that is different, or the same. Perhaps serving the /meta data. 

I wouldn't mess with CORS origins, unless you have access to the AWS account that setup the stack, so if you do need to delete the Parameter Store cors_origins, you can.

I made a bug on hubs-cloud to investigate what you're seeing with Google Analytics: https://github.com/mozilla/hubs-cloud/issues/117

**Reply by robinkwilson** [reply]
Added a **FOR NOW** solution in the admin panel: https://github.com/mozilla/hubs-cloud/issues/117

**Reply by robinkwilson** [reply]
Solved will be pushed out to HC soon. https://github.com/mozilla/reticulum/pull/420
